src/lib/utils.ts

```diff
@@ -13,6 +13,16 @@ export function isBuffer(obj: any) {
   );
 }
 
+/**
+ * Converts GitHub blob URLs to raw URLs
+ */
+export function normalizeUrl(url: string) {
+  if (url.startsWith('https://github.com/')) {
```

Comment:
Should we also look for `/blob/`?

Reply:
I feel like if a URL starts with `https://github.com/` and doesn't contain `/blob/`, it's probably going to fail anyways? But doesn't hurt to have so added in https://github.com/readmeio/oas-normalize/pull/236/commits/baf785b76b5c8bc23eed23d96b03f4430eb9ab19